harvester_e2e_tests/apis/test_volumes.py

```diff
@@ -119,3 +154,46 @@ def test_delete(self, api_client, unique_name, wait_timeout):
                 f"Failed to delete volume {unique_name} with {wait_timeout} timed out\n"
                 f"Still got {code} with {data}"
             )
+
+    def test_create_volume_backing_image(self, api_client, unique_name, opensuse_image):
```

Comment:
let's move the test case into `integration/test_volumes.py`
the file contains some legacy test cases might need to be removed or rewrite.

Reply:
Changed